pkg/apis/core/validation/shoot.go

```diff
@@ -878,6 +878,17 @@ func ValidateClusterAutoscaler(autoScaler core.ClusterAutoscaler, fldPath *field
 		allErrs = append(allErrs, field.NotSupported(fldPath.Child("expander"), *expander, availableClusterAutoscalerExpanderModes.List()))
 	}
 
+	if ignoreTaints := autoScaler.IgnoreTaints; ignoreTaints != nil {
+
+		k8sVersionIs116OrGreater, _ := versionutils.CompareVersions(k8sVersion, ">=", "1.16")
+
+		if k8sVersionIs116OrGreater {
+			allErrs = append(allErrs, isIgnoreTaintsUnique(ignoreTaints, fldPath.Child("ignoreTaints"))...)
+		} else {
+			allErrs = append(allErrs, field.Forbidden(fldPath.Child("ignoreTaints"), "is not support for k8s version < 1.16"))
```

Comment:
Having in mind https://github.com/gardener/gardener/issues/3470, I guess `field.Invalid` would be more appropriate than `field.Forbidden`.

```suggestion
			allErrs = append(allErrs, field.Invalid(fldPath.Child("ignoreTaints"), autoScaler.IgnoreTaints, "ignoreTaints cannot be specified for kubernetes version < 1.16"))
```